feature/achievements/src/main/java/io/github/droidkaigi/confsched2023/achievements/AchievementsScreen.kt

```diff
@@ -138,7 +140,7 @@ fun AchievementScreenDialog(
             )
         },
         text = {
-            Column {
```

Comment:
```
val scrollState = rememberScrollState()
```

When considering the reuse of scrollState, you might find it helpful to store `rememberScrollState()` in a variable, much like what's suggested in the official documentation.